tekton/release-full-setup.md

```diff
@@ -0,0 +1,109 @@
+# Release Pipeline
+
+## Setup
+
+> **Note:**
+> Depending on your cluster you may need to increase the memory allocated. 6GB is known to work on Docker Desktop.
+> See https://github.com/tektoncd/pipeline/issues/2417 for details.
```

Comment:
Let's remove this, I've been running with 3 or 4GB for a year with no problems, this was a very old issue.